README.md

```diff
@@ -188,7 +188,6 @@ Environment files are documented [here](./ENVIRONMENT.md), and a sample environm
 1. Copy the template values into the .env files.
    ```sh
    cp env.template .env
-   cp env.template .env.dev
```

Comment:
why is this gone? assuming we don't have this file anymore?

Reply:
Correct. I don't think anything uses `.env.dev` anymore